ClinGen/clincoded
# Connects to #1095. GCI score implementation fixes.

**Technical notes:**
This PR consists of additional changes and fixes for the scoring of Individual, Experimental and Case-Control evidence.

Some of the issues addressed by this PR can be referenced in the subsequent comments in #1138 (https://github.com/ClinGen/clincoded/issues/1138#issuecomment-267128332).

**Steps to test:**
1. In the score panel and if a score explanation is required, expect to see the **'required'** asterisk character to be on the right side of the score explanation label instead of being underneath it.
![image](https://cloud.githubusercontent.com/assets/6956310/21244087/cb4cd8a2-c2d0-11e6-807b-5661b3ac616b.png)

2. If a score explanation is required but no explanation has been provided in the textarea, when clicking on the **'Save'** button, expect to see the form **not** being submitted and a note is displayed to alert the user.
![image](https://cloud.githubusercontent.com/assets/6956310/21246904/62be96a8-c2e0-11e6-9bca-e756bca2802c.png)

3. On the Experimental curation form and viewer pages, expect to see **no** assessments UI being shown unless the evidence has existing assessment data.
4. On the Individual curation form page, if the MOI is neither AD, AR or X-Linked, select the 'Supports' or 'Review' score status and expect to see all 5 Case Information types in the dropdown but no additional scoring options.
![image](https://cloud.githubusercontent.com/assets/6956310/21244169/244e7b36-c2d1-11e6-9fb0-45abddca82f7.png)

5. For 3 different types of evidence, expect to see that other users can't score unless the evidence-submitter had already scored it first.
